windows/src/developer/TIKE/child/UfrmDebug.pas

```diff
@@ -1254,116 +1322,32 @@ function TfrmDebug.ShortcutDisabled(Key: Word): Boolean;
     Result := True
   else if GetKeyState(VK_CONTROL) < 0 then
   begin
-    if Key in [Ord('A').. Ord('Z'), Ord('0')..Ord('9')] then
+    if Key in [Ord('A')..Ord('Z'), Ord('0')..Ord('9')] then
       Result := True;
   end;
 end;
 
 procedure TfrmDebug.memoSelMove(Sender: TObject);
-var
-  ch: WideString;
 begin
   if memo.Focused then
   begin
     frmKeymanDeveloper.barStatus.Panels[0].Text := 'Debugger Active';
-    if memo.SelText = ''
-      then ch := Unicode.CopyChar(memo.Text, memo.SelStart-1, 1)
-      else ch := Copy(memo.SelText, 1, 16);
-
-    if ch = ''
-      then frmKeymanDeveloper.barStatus.Panels[2].Text := ''
-      else frmKeymanDeveloper.barStatus.Panels[2].Text := FormatUnicode(ch);
```

Comment:
The character grid provides this data, no need to also have a poorer version of it in the status bar.